internal/planner/selector.go

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package planner
+
+import (
+	"encoding/json"
+)
+
+// BuiltinDefaultNodeSelector defines our defaults if no administrative
+// overrides are supplied. Samba containers run only on linux. Currently
+// they're only built for x86_64 (amd64).
+var BuiltinDefaultNodeSelector = map[string]string{
```

Comment:
Could be private var (rename to `builtinDefaultNodeSelector` or just `defaultNodeSelector`).

Reply:
Sure.

Reply:
Resolved.